pkg/commands/configstoreentry/errors.go

```diff
@@ -8,8 +8,11 @@ import (
 )
 
 const (
-	maxKeyLen   = 256
-	maxValueLen = 8000
+	maxKeyLen = 256
+	// maxValueLen is the maximum length of config store entry's value. It's set to 64k,
+	// even though users may have a smaller limit. The API will reject requests if the
```

Comment:
By 'user' here do you mean 'customer' (as in `CID`)? If so, the comment should probably be specific about that, so there's no confusion.